.github/workflows/deploy-oss-catalog.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy OSS Connector Catalog to GCS
+
+on:
+  push:
+    branches:
+      - master
+    paths: 
+      - airbyte-config/init/src/main/resources/seed
+
+  workflow_dispatch:
+
+jobs:
+  deploy-catalog:
+    name: "Deploy Catalog"
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    concurrency: deploy-oss-connector-catalog
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v2
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.PROD_SPEC_CACHE_SA_KEY }}
+          export_default_credentials: true
+      - name: Generate catalog
+        run: SUB_BUILD=PLATFORM ./gradlew :airbyte-config:init:processResources
+      - name: Upload catalog to GCS
+        shell: bash
+        run: |
+          gcs_bucket_name="prod-airbyte-cloud-connector-metadata-service"
```

Comment:
This isn't a secret, right?

Reply:
Should there be an OSS bucket that's distinct from the Cloud bucket?

Reply:
These files are meant to be public so I think it's ok for the bucket name to be in plaintext. The spec cache bucket name is also in the publish scripts.

As for having a separate bucket, infra said it was ok for them to be in the same bucket, and I think it might make things easier down the line if we want to actually put these behind a domain if they're in the same bucket.

Reply:
👍 then!

I guess ideally the bucket name wouldn't have had ☁️ in the name... but 🤷

Reply:
Agree 😅